transformers/events/median_lift/median_lift_suite_test.go

```diff
@@ -0,0 +1,19 @@
+package median_lift
+
+import (
+	"io/ioutil"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+)
+
+func TestMedianLift(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "MedianLift Suite")
+}
+
+var _ = BeforeSuite(func() {
```

Comment:
nbd but I don't _think_ we need this anymore if we're not logging in the transformer